service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaSubscriberActor.scala

```diff
@@ -122,11 +126,24 @@ private[lagom] class KafkaSubscriberActor[Message](
 
 object KafkaSubscriberActor {
   def props[Message](
-    kafkaConfig: KafkaConfig, consumerConfig: ConsumerConfig, locateService: String => Future[Option[URI]],
-    topicId: String, flow: Flow[Message, Done, _],
-    consumerSettings: ConsumerSettings[String, Message], subscription: AutoSubscription,
-    streamCompleted: Promise[Done]
+    kafkaConfig:    KafkaConfig,
```

Comment:
@TimMoore, I added some extra formatting like this one in a few places. It's very hard to have a clear picture of all the values being passed to a constructor when the list gets long and we need to parse the code looking for colons. 

A break line is a much nicer way to help the human eye to recognise different items.